Source/Particles/Deposition/CurrentDeposition.H

```diff
@@ -656,7 +659,10 @@ void doEsirkepovDepositionShapeN (const GetParticlePosition<PIdx>& GetPosition,
                                   const amrex::XDim3 & xyzmin,
                                   amrex::Dim3 lo,
                                   amrex::Real q,
-                                  [[maybe_unused]]int n_rz_azimuthal_modes)
+                                  [[maybe_unused]]int n_rz_azimuthal_modes,
```

Comment:
Not important formatting of existing code:
```suggestion
                                  [[maybe_unused]] int n_rz_azimuthal_modes,
```